aws_lambda_powertools/utilities/parameters/base.py

```diff
@@ -35,7 +37,9 @@
     from mypy_boto3_ssm import SSMClient
 
 
-DEFAULT_MAX_AGE_SECS = 5
+# If the environment variable is not set, the default value is 5
+DEFAULT_MAX_AGE_SECS: int = int(os.getenv(constants.PARAMETERS_MAX_AGE, 5))
+
```

Comment:
I'm note sure if we should create a function to check if it is integer or not. Currently, we do not control whether `max_age` is an integer anywhere in the code.